client/my-sites/checkout/checkout-thank-you/marketplace/style.scss

```
@@ -87,16 +85,21 @@
 				line-height: 20px;
 				font-size: $font-body-small;
 				color: var(--studio-gray-60);
+				margin-bottom: 16px;
 			}
 		}
 
 		.thank-you__step-cta {
 			margin: 0;
-		}
 
-		.thank-you__step-icon {
-			width: calc(18px + 23px); // icon size + padding
-			flex-basis: 100%;
+			a {
+				font-weight: 500;
+				font-size: $font-body;
```

According to the design, I think this should be
```suggestion
				font-size: $font-body-small;
```

---

Great points @epeicher, thank you for the thorough review! 🙏